plugins/xen/xen-multi

```diff
@@ -146,13 +151,16 @@ while (<XENTOP>) {
 
   # We need the remaining data only for a normal run
   if ($ARGV[0] eq "") {
-    $domains{$domain}{'cpusecs'} = $cpusecs;
-    $domains{$domain}{'cpupercent'} = $cpupercent;
-    $domains{$domain}{'mem'} = $memk;
-    $domains{$domain}{'nettx'} = $nettxk;
-    $domains{$domain}{'netrx'} = $netrxk;
-    $domains{$domain}{'vbdrd'} = $vbdrd;
-    $domains{$domain}{'vbdwr'} = $vbdwr;
+    $domains{$domain}{'cnt'}++;
```

Comment:
Maybe you should use a separate variable (outside of this hash) in order to keep the code more readable?

Reply:
The cnt key must be stored per domain and the $domain hash just has exactly the same data structure, so IMHO it is more readable using the $domain variable than making a new variable for it.